packages/@sanity/migrate/src/defineMigration.ts

```diff
@@ -1,5 +1,29 @@
 import {type Migration} from './types'
 
+/**
+ * Helper function for defining a Sanity content migration. This function does not do anything on its own;
+ * it exists to check that your schema definition is correct, and help autocompletion in your IDE.
+ *
+ * @see https://www.sanity.io/docs/schema-and-content-migrations#af2be129ccd6
```

Comment:
`@see` is not support need to use `@link`

```suggestion
 * {@link https://www.sanity.io/docs/schema-and-content-migrations#af2be129ccd6}
```